build.gradle

```diff
@@ -1034,6 +1034,13 @@ task releaseAndTest {
 // Don't create an empty checker-framework-VERSION.jar
 jar.onlyIf {false}
 
+clean.doLast {
```

Comment:
This should probably be in `checker/build.gradle` and use `buildDir.exists()` rather that `file("...").exists()`